src/plugins/folderView/FolderGridView.js

```diff
@@ -33,7 +33,7 @@ define([
             name: 'Grid View',
             cssClass: 'icon-thumbs-strip',
             canView: function (domainObject) {
-                return domainObject.type === 'folder';
+                return domainObject.type === 'folder' || domainObject.type === 'noneditable.folder';
```

Comment:
Same comment as below about having an array of types

Reply:
Added a constants file to use in both.